src/picture.rs

```diff
@@ -42,10 +40,21 @@ use std::ffi::c_void;
 use std::io;
 use std::ptr;
 
-pub type PictureFlags = c_uint;
-pub const PICTURE_FLAG_NEW_TEMPORAL_UNIT: PictureFlags = 4;
-pub const PICTURE_FLAG_NEW_OP_PARAMS_INFO: PictureFlags = 2;
-pub const PICTURE_FLAG_NEW_SEQUENCE: PictureFlags = 1;
+bitflags! {
+    #[derive(Clone, Copy, PartialEq, Eq, Hash, Default)]
+    pub struct PictureFlags: u8 {
+        const NEW_SEQUENCE = 1 << 0;
+        const NEW_OP_PARAMS_INFO = 1 << 1;
+        const NEW_TEMPORAL_UNIT = 1 << 2;
+    }
+}
+
+impl From<PictureFlags> for Rav1dEventFlags {
+    fn from(value: PictureFlags) -> Self {
+        // [`Rav1dEventFlags`] just has one extra flag.
+        Self::from_bits_truncate(value.bits())
```

Comment:
What happens here if picture flags has new_temporal_unit?

Reply:
Looks like it should be ignored, but I don't know the bitflags API off the top of my head. If so, maybe just add a comment about that?

Reply:
> What happens here if picture flags has new_temporal_unit?

It'll get truncated off (the same behavior of the code before).

I'll explain it in the comment.  I was trying to but I don't think I did a very good job of it.

Reply:
Added in 928f4a8.